src/backend/api/downloadmanager.ts

```diff
@@ -14,12 +14,15 @@ export const install = async (args: InstallParams) => {
   ipcRenderer.invoke('addToDMQueue', dmQueueElement)
 
   // Add Dlcs to the queue
-  if (Array.isArray(args.installDlcs) && args.installDlcs.length > 0) {
+  if (
+    Array.isArray(args.installDlcs) &&
+    args.installDlcs.length > 0 &&
+    args.runner === 'legendary'
```

Comment:
we don't want to add DLCs to the download queue if all are selected for GOG? is it that it works differently than legendary?

Reply:
For GOG we do everything in one go since manifests are not separate for each product like for Epic